xoops_lib/Xoops/Form/Renderer/Bootstrap3Renderer.php

```diff
@@ -191,9 +191,9 @@ protected function renderXoopsFormDhtmlTextArea(\Xoops\Form\DhtmlTextArea $eleme
         // fonts
         $ret .= $element->typographyControls();
 
-        // the textarea box
-
+        // the textarea box		
         $element->suppressRender(['value']);
+		$element->set('class', 'form-control');
```

Comment:
Should this be $element->add('class', 'form-control'); instead?

The set() replaces whatever is there, while add() adds a new class to whatever is already there there.

Reply:
You're right, I was not sure